package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "chai-bignumber": "^2.0.2",
+    "chai-as-promised": "^7.0.0",
```

Comment:
we've removed this dependency for various reasons; if you're using promise assertions in the test, you can update them to not use this library.

Reply:
I've removed it and edited test script.